src/PackageCompiler.jl

```diff
@@ -834,7 +843,8 @@ function create_app(package_dir::String,
                     sysimage_build_args::Cmd=``,
                     include_transitive_dependencies::Bool=true,
                     include_preferences::Bool=true,
-                    script::Union{Nothing, String}=nothing)
+                    script::Union{Nothing, String}=nothing,
+                    precompile_execution_args::Cmd=`-O0`)
```

Comment:
Can we split the difference and do -O1?

Reply:
Seems plausible to me, but what is that based on, @sjkelly?  My gut is that -O0 would be better for almost all snoop workloads, since the developer is already pretty strongly incentivized to keep data sizes small, and thus to have compilation-dominated workloads.

What leads you to prefer -O1?